ToDo-Garden/ToDoGardenUI/Sources/ToDoGardenUIComponent/UISwitch+ToDoGardenSwitch.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  UISwitch+ToDoGardenSwitch.swift
```

Comment:
```suggestion
//  ToDoGardenSwitch.swift
```
`Class`로 재정의 해주신 것 확인했습니다! 파일명이 바뀌어야하지않을까요?

Reply:
변경완료 하였습니다 
>[2c1a37a](https://github.com/ToDo-Garden/ToDo-Garden/pull/41/commits/2c1a37a9b9e77ea83d8ebddcc4427197ed89a835)